### PR TITLE
Fix po and pot file

### PR DESCRIPTION
**Pull request by hliebscher**
found a bug that not create new pot file from messages
in CMakeList
add_custom_target(gettext_make_pot
    COMMAND xgettext --keyword=L --keyword=_L --keyword=_u8L --keyword=L_CONTEXT:1,2c --keyword=_L_PLURAL:1,2 --add-comments=TRN --from-code=UTF-8 --no-location --debug --boost
        -f "${BBL_L18N_DIR}/list.txt"
        -o "${BBL_L18N_DIR}/BambuStudio.pot"
but manual it works

add src/slic3r/GUI/calib_dlg.cpp in list.txt for missing messages

then create a net Softfever pot file and merge with po

xgettext --keyword=L --keyword=_L --keyword=_u8L --keyword=L_CONTEXT:1,2c --keyword=_L_PLURAL:1,2 --add-comments=TRN --from-code=UTF-8 --no-location --debug --boost   -f "bbl/i18n/list.txt" -o "bbl/i18n/BambuStudio-Softfever.pot"

msgmerge bbl/i18n/de/BambuStudio_de.po bbl/i18n/BambuStudio-Softfever.pot -o bbl/i18n/de/BambuStudio_de.po 

and translate the messages

### CODE REVIEW

**Comment by SoftFever** [comment]
Thanks. 
Will test it before merging.
I must admit I don't know much about the po/pot/mo thing -_-

**Comment by hliebscher** [comment]
gettext is a library and tool that allows developers to translate their applications into multiple languages. It provides functions for translating text within the application and managing the translations in PO (Portable Object) files.

PO files contain the translations for the texts within the application, as well as information such as the original text, translation, translator, and translation status.

msgmerge is a tool used to make changes to existing PO files, such as adding new texts or making changes to existing translations without affecting the work of the translators. It allows developers to automatically add new texts added to the application to the PO files or synchronize existing PO files with updated original texts.

POT files are template files used to create new PO files containing the texts used in the application.

In summary, gettext is a tool for making translations in applications, it uses PO files which contains the translations and msgmerge and POT files are used to update the PO files and create new PO files.

**Comment by SoftFever** [comment]
> gettext is a library and tool that allows developers to translate their applications into multiple languages. It provides functions for translating text within the application and managing the translations in PO (Portable Object) files.
> 
> PO files contain the translations for the texts within the application, as well as information such as the original text, translation, translator, and translation status.
> 
> msgmerge is a tool used to make changes to existing PO files, such as adding new texts or making changes to existing translations without affecting the work of the translators. It allows developers to automatically add new texts added to the application to the PO files or synchronize existing PO files with updated original texts.
> 
> POT files are template files used to create new PO files containing the texts used in the application.
> 
> In summary, gettext is a tool for making translations in applications, it uses PO files which contains the translations and msgmerge and POT files are used to update the PO files and create new PO files.

Cool.
What does the list.txt do here?

**Comment by hliebscher** [comment]
list.txt has all files who are to translated messages located and scan word=_u8L --keyword=L_CONTEXT:1,2c --keyword=_L_PLURAL:1,2   if message with L("????  ths copy this message to pot file under msgid=

**Comment by SoftFever** [comment]
> 

Thanks for the explanation.
Now I know how the xgettext works.
Appreciate it!
